Modules/pyexpat.c

```diff
@@ -76,6 +76,7 @@ typedef struct {
     PyObject_HEAD
 
     XML_Parser itself;
+    PyObject *parent;           /* Parent xmlparseobject (for ref counting) */
```

Comment:
```suggestion
    /*
     * Strong reference to a parent `xmlparseobject` if this parser
     * is a child parser. Set to NULL if this parser is a root parser.
     * This is needed to keep the parent parser alive as long as it has
     * at least one child parser.
     *
     * See https://github.com/python/cpython/issues/139400 for details.
     */
    PyObject *parent;
```

(I personally prefer comments above as it's easier to update them rather than aligned ones when the text is too long).

Reply:
@picnixz it's very "into the face" then though, it hijacks the whole structure. I can add it but are you sure? I'll add it but it feels like too much.

Reply:
It's fine. It's a non-trivial reason why we have it and I prefer being explicit here since the project is touched by many people.